src/main/java/pl/debuguj/parkingspacessystem/services/PaymentServiceImpl.java

```diff
@@ -0,0 +1,87 @@
+package pl.debuguj.parkingspacessystem.services;
+
+import org.joda.time.DateTime;
+import org.joda.time.Minutes;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import pl.debuguj.parkingspacessystem.domain.ParkingSpace;
+import pl.debuguj.parkingspacessystem.enums.Currency;
+
+import java.math.BigDecimal;
+
+/**
+ * Created by grzesiek on 10.10.17.
+ */
+@Service
+public class PaymentServiceImpl implements PaymentService {
```

Comment:
to nie musi być publiczna klasa. niepotrzebny interfejs

Reply:
Tak wszystkie klasy w tym projekcie moga być prywatne. Intefejsy robię z nawyku, być może to zły nawyk.